pkg/jaegerquery/get_services.go

```diff
@@ -0,0 +1,38 @@
+// This file and its contents are licensed under the Apache License 2.0.
+// Please see the included NOTICE for copyright information and
+// LICENSE for a copy of the license.
+
+package jaegerquery
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/jackc/pgtype"
+	"github.com/jackc/pgx/v4"
+	"github.com/pkg/errors"
+	"github.com/timescale/promscale/pkg/pgxconn"
+)
+
+const getServicesSQL = `
+SELECT
+ 	array_agg(value#>>'{}' ORDER BY value)
```

Comment:
Just curious, why we call `array_agg`? I guess we can do the same without it

Reply:
returning and populating arrays is more efficient than returning rows in our performance tests.